services/security/certificate/certificate.go

```diff
@@ -32,3 +34,45 @@ func getWssdCertificate(cert *security.Certificate) (*wssdcloudsecurity.Certific
 		Name: *cert.Name,
 	}, nil
 }
+
+func getWssdCSR(csr *security.CertificateRequest) (*wssdcloudsecurity.CertificateSigningRequest, string, error) {
```

Comment:
Try to use Moc instead of Wssd.
We will rename in other places, but you can start using Moc.